CHANGES.md

```diff
@@ -35,9 +39,13 @@ Format of the entries must be.
 
 * Mesos does not expose ZooKeeper credentials anymore via its state JSON document. (DCOS_OSS-2162)
 
+* TLS: Admin Router should be configured with both RSA and EC type certificates. (DCOS-22050)
```

Comment:
@mhrabovcin  - Could you please verify if this changes.md addition is good for your changes you are bringing in with https://github.com/dcos/dcos/pull/2746 ?

In future, you could add these CHANGES.md addition in your PR too.

Reply:
@mhrabovcin  - I think, we should have a Open JIRA (DCOS_OSS-XXX) here for the dcos/dcos CHANGES.md file. WDYT? If you have one, please point it to here, otherwise you could clone the above JIRA and share it. TIA.